### ISSUE TITLE
Worlds with Astral Sorcery not showing with correct skybox

### ISSUE THREAD

**Issue by EnderCorpsAlpha** [Abastro/StellarSky#99]
Sorry to possibly write a duplicate thread, But I noticed the other one had an invalid tag after having a lack of info about it.

The issue is this - When Astral sorcery and Stellarsky are both installed into the same instance only the vanilla skybox with the astral sorcery augments shows - I think this might be down to how AS deals with the constellations in the mod but I'm not entirely sure.

I'd love to include this mod in a pack I'm tinkering around with, But I can't add it due to this incompatibility

**Comment by Abastro** [comment]
I don't know what's happening here, it could be complex compatibility issue.
Before doing anything, I recall that there was a setting in Astral Sorcery to enable other skybox mods to render its skybox. Did you try it?

**Comment by EnderCorpsAlpha** [comment]
Yeah, Sorry I wrote this thread before I knew that option existed. It worked and StellarSky's skybox is now showing, but no one can see the constellations from AS now, So not sure on that front.

**Comment by Abastro** [comment]
Hmm, that's strange. I'll investigate this.

**Comment by EnderCorpsAlpha** [comment]
I think it's because AS replaces the vanilla skybox completely to do the constellations, Though I don't understand why they just don't overlay it on - Also viewing constellations through the telescope doesn't seem to work either - I'll say to the AS Dev about this too

**Comment by ttamttamS** [comment]
Try turning on weak sky render (or something like that, i forget the exact name) in astral sorcery config

**Comment by LordGeo** [comment]
i have a similar problem everything was fine until i did the thing with the telescopes and the constellations and now everything goes black at nigth. i tried the weakskyrender option but it didnt work. i have sildur  shaders and optifine and some other mods installed. any suggestions on what to do

**Comment by EnderCorpsAlpha** [comment]
I'd put that on Optifine and the Shaders - The shaders more so. That's because most shader packs I know of screw around with the skybox and Optifine messes around with that kind of thing too. Tbh I've found optifine slows down modded clients for me more than not actually having it

**Comment by Xorbah** [comment]
Was there ever a workaround found for this?

**Comment by KiriCattus** [comment]
Apologies but we have yet to find a dev to revive work on the stellar sky mods since abastro retired a year back maybe but as soon as we do you’ll likely see some activity in the repo.

Those of us who are on the team so far have not had the time or skill to be able to bring it back fully but we are still looking
